tests/unit/panther_analysis_tool/test_rule.py

```diff
@@ -0,0 +1,694 @@
+# Copyright (C) 2020 Panther Labs Inc
+#
+# Panther Enterprise is licensed under the terms of a commercial license available from
+# Panther Labs Inc ("Panther Commercial License") by contacting contact@runpanther.com.
+# All use, distribution, and/or modification of this software, whether commercial or non-commercial,
+# falls under the Panther Commercial License to the extent it is permitted.
+
+import dataclasses
+import json
+import inspect
+import os
+import pathlib
+import shutil
+from typing import Tuple, Optional, Any
+from unittest import TestCase
+import tempfile
+from types import ModuleType
+
+from panther_analysis_tool.rule import MAX_DEDUP_STRING_SIZE, MAX_GENERATED_FIELD_SIZE, \
+    Rule, RuleResult, TRUNCATED_STRING_SUFFIX, FilesystemImporter, RawStringImporter
+from panther_analysis_tool.enriched_event import PantherEvent
+
+
+class TestRule(TestCase):  # pylint: disable=too-many-public-methods
```

Comment:
Awesome! Now all we need is some way to synchronize changes from the rules-engine to this

Reply:
It is the other way around 😄 , the Rules Engine depends on these modules now. You can make a change here and it will be available in the Rules Engine too.